src/buffer/out/CharRowCell.hpp

```diff
@@ -43,9 +43,8 @@ class CharRowCell final
 
     friend constexpr bool operator==(const CharRowCell& a, const CharRowCell& b) noexcept;
 
-private:
-    wchar_t _wch;
-    DbcsAttribute _attr;
+    wchar_t _wch{ UNICODE_SPACE };
+    DbcsAttribute _attr{};

```

Comment:
these two became public -- intended?

Reply:
This is leftover from trying to do more POD types - I'll make these private again, thanks for catching.